src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -463,12 +463,16 @@ def backup_now(cmd, client, resource_group_name, vault_name, item, retain_until,
     message = "For SAPHANA and SQL workload, retain-until parameter value will be overridden by the underlying policy"
 
     if retain_until is None:
-        retain_until = datetime.now(timezone.utc) + timedelta(days=30)
-        if backup_type == 'Full':
+        if backup_type.lower() == 'copyonlyfull':
```

Comment:
Could you ensure that the value of `backup_type` is not `None`, otherwise exception may be caused here

Reply:
We have a `if backup_type is None:` check above in this method already

Reply:
Got it, thanks~